source/Body.cpp

```diff
@@ -345,9 +345,11 @@ Body::initiateStep(vec6 r, vec6 rd)
 		r_ves = vec6::Zero();
 		rd_ves = vec6::Zero();
 		return;
+	} else {
```

Comment:
Why is this needed? The returns are already short-cutting the function execution

Indeed this is wrong, since the else only affects to [the second if](https://github.com/FloatingArrayDesign/MoorDyn/blob/v2/source/Body.cpp#L343)

Reply:
You are right, I think this must have been an accidental change on my part. I've returned it to the way it was previously.